kiosk/src/Components/MainMenu.tsx

```diff
@@ -43,13 +50,16 @@ const MainMenu: React.FC<IProps> = ({ kiosk }) => {
         ) {
             tickEvent("kiosk.addGamePageLoaded");
             kiosk.launchAddGame();
+            playSoundEffect("select");
         }
         if (
             deleteButtonSelected &&
             (kiosk.gamepadManager.isAButtonPressed() ||
                 kiosk.gamepadManager.isBButtonPressed())
         ) {
+            kiosk.gamepadManager.blockAPressUntilRelease();
```

Comment:
Would we want to block for other cases as well? Was the problem that if people selected delete game and with the increased polling that the pressed would stay triggered and just cancel the delete modal really quickly?

Reply:
I wanted to be careful about where to add this. There may be a few more places where it's useful. It's a short term fix. I have a new game controller navigation lib incoming that handles this a different way.